temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,114 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message IncomingServiceSpec {
+    // The name of the incoming service. Must be unique within an account
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    string name = 1;
+
+    // The namespace to route requests to
+    string namespace = 2;
+
+    // The task queue to route requests to
+    string task_queue = 3;
+
+    // The set of authorization policies for the service. Each request's caller
+    // must match with at least one of the specs to be accepted by the incoming service
+    repeated IncomingServiceAuthorizationSpec authorization_spec = 4;
+}
+
+message IncomingServiceAuthorizationSpec {
+    // The policy set on this incoming service authorization spec
+    // Possible values are: ALLOWED_CLOUD_NAMESPACE
+    string policy_type = 1;
+
+    // A policy spec that allows one caller namespace to access the service
+    AllowedCloudNamespacePolicySpec allowed_cloud_namespace_policy_spec = 2;
+}
+
+message AllowedCloudNamespacePolicySpec {
+    // The namespace that is allowed to call into this service. Calling namespace must be in same account as incoming service
+    string namespace = 1;
```

Comment:
Just to be clear, this is the complete namespace string (abhinavns1.a2dd6) and not just namespace name (abhinavns1)?

If yes, we won't need to change a thing when we start supporting cross account services right?

Reply:
yes, this is the complete namespace string